src/main/java/prgms/spring_week1/menu/VoucherMenu.java

```diff
@@ -0,0 +1,23 @@
+package prgms.spring_week1.menu;
+
+import java.util.stream.Stream;
+
+public enum VoucherMenu {
+    INSERT("insert"),
+    FIND_ALL("findall"),
+    FIND_BY_TYPE("findbytype"),
+    DELETE_ALL("deleteall");
```

Comment:
아래의 형태가 관리하기 더 값을 추가, 삭제하는데 더 편하실 거에요 :) 
```suggestion
    INSERT("insert"),
    FIND_ALL("findall"),
    FIND_BY_TYPE("findbytype"),
    DELETE_ALL("deleteall")
    ;
```

Reply:
5b08cb6ac741e5742f3cd8818e96953321163c2d 커밋 피드백 반영
- 넵 해당 형식으로 수정하였습니다 !